src/functionalTest/java/uk/gov/hmcts/st/taskconfiguration/FunctionalTest.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.st.taskconfiguration;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class FunctionalTest {
+    @Value("${test.url:http://localhost:4551}")
+    private String testUrl;
+
+    @Test
+    public void testHealthEndpoint() {
```

Comment:
Just a small thing - is it worth exporting this method to SmokeTest.java as they're the same? Or vice versa?